### PR TITLE
Name the artifact with the tag

### PR DESCRIPTION
**Pull request by ocefpaf**
Let's see if this works. Addresses https://github.com/introocean/introocean-en/pull/7#issuecomment-1260084090 from @sergeykorop.

### CODE REVIEW

**Comment by ocefpaf** [comment]
It will still have a generic name "book" on PR artifacts but those should be used only to check the build. When tagging the pdf will be named `tag-name.pdf`. You can see a test in my fork in https://github.com/ocefpaf/introocean-en/releases/tag/v20220922

File: .github/workflows/publish_book.yml [hunk 1]
```diff
@@ -27,11 +27,16 @@ jobs:
       with:
         path: compiled_book/
 
+    - name: Rename book to tag
+      if: startsWith(github.ref, 'refs/tags/')
+      shell: bash -l {0}
+      run: mv compiled_book/book.pdf compiled_book/${{ github.ref_name }}.pdf
```
Comment (sergeykorop) [inline .github/workflows/publish_book.yml:34]: Given the old good bash I would use something like

`"compiled_book/introocean-en-${REF_NAME:1}.pdf"`

where `REF_NAME` is an environment variable keeping the tag name to get a release name with a prefix and remove the starting char from the tag name (assuming tag name follows the convention `v20220922`).

Not sure, however, that such syntax is allowed for this config file format.

Comment (ocefpaf) [reply .github/workflows/publish_book.yml:34]: Should work. Want to give it a try and sent a commit to implement that?

**Comment by koldunovn** [comment]
Is it ready to be merged?

**Comment by ocefpaf** [comment]
> Is it ready to be merged?

IMO, yes.

**Pull request merged**